index.html

```diff
@@ -123,6 +125,93 @@ <h3 class="title-1">Multi-Post <br> Stories</h3>
             </div>
         </div>
     </section>
+    <section class="desk-works">
+        <div class="works-card">
```

Comment:
- [ ] On the Figma design's cards it seems there is this little gray border around the cards, and likewise to the images in the card, Kindly let us fix this 😀.